packages/react-components/react-popover/src/components/Popover/Popover.types.ts

```diff
@@ -87,6 +87,13 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
    */
   openOnHover?: boolean;
 
+  /**
+   * Flag to close the Popover when an iframe outside a PopoverSurface is focused
+   *
+   * @default false
+   */
+  closeOnIframeFocus?: boolean;
```

Comment:
AFAIK the Menu and compat Datepicker also use this utility, should the prop be implemented in those components too?

Reply:
Yes, these components use this util. We will implement this prop based on requests in future.